src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 20px;
+
+  gap: $gap-size;
+  margin: auto;
+  width: getWidth($gap-size, $day-size);
+
+  @for $i from 28 through 31 {
+    &-month-length--#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+        &::before {
+          content: "#{$i}";
+          font-family: Arial, Helvetica, sans-serif;
+          font-size: 30px;
+        }
+      }
+    }
+  }
+
+  &__day:first-of-type {
+    margin-left: (6 * $day-size) + (5 * $gap-size);
+  }
+
+  &__day:hover {
+    transform: translateY(-20px);
```

Comment:
these properties are to be animated with the duration of half a second

Reply:
Yes, color and movement should be transitioned.